applications/nrf_desktop/src/events/ble_event.h

```diff
@@ -59,6 +59,19 @@ enum peer_operation {
 	PEER_OPERATION_COUNT
 };
 
+/** @brief Peer type list. */
+#define PEER_TYPE_LIST			\
+	X(MOUSE)			\
+	X(KEYBOARD)			\
+
+/** @brief Peer types (position in bitmask). */
+enum peer_type {
+#define X(name) _CONCAT(PEER_TYPE_POS_, name),
```

Comment:
remove `POS` suffix
`s/PEER_TYPE_POS_/PEER_TYPE_`

Reply:
I would prefer to leave it. It is used as position in bitmask when defining peer type (it does not simply represent peer type)